README.md

```diff
@@ -340,5 +340,60 @@ catalog:
 
 Check out the [examples](examples/hive) and the [E2E test](e2e/hive)
 
+
+#### Apache Hudi
+Metorikku supports reading/writing with [Apache Hudi](https://github.com/apache/incubator-hudi).
+
+Hudi is a very exciting project that basically allows upserts and deletes directly on top of the data lake.
+
+In order to use Hudi with Metorikku you need to add to your classpath (via ```--jars``` or if running locally with ```-cp```) 
+an external JAR from here: https://github.com/YotpoLtd/incubator-hudi/releases/download/hoodie-0.4.5-metorikku/hoodie-spark-bundle-0.4.5-hive-unshaded.jar
+
+After that you can start using the new Hudi writer like this:
+
+#### Job config
+```yaml
+output:
+  hudi:
+    dir: /examples/output
+    # This controls the level of parallelism of hudi writing (should be similar to shuffle partitions)
+    parallelism: 1
+    # upsert/insert
+    operation: upsert
+    # COPY_ON_WRITE/MERGE_ON_READ
+    storageType: COPY_ON_WRITE
+    # Maximum number of versions to retain 
+    maxVersions: 1
+    # Hive database to use when writing
+    hiveDB: default
+    # Hive server URL
+    hiveJDBCURL: jdbc:hive2://hive:10000
+    hiveUserName: root
+    hivePassword: pass
+```
+
+#### Metric config
+```yaml
+dataFrameName: test
+  outputType: Hudi
+  outputOptions:
+    path: test.parquet
+    # The key to use for upserts
+    keyColumn: userkey
```

Comment:
compound key ?

Reply:
It's called key column by hudi, I tried to keep it similar, but it can be compound of course

Reply:
how can I separate it ? can I write for example: id,name,email ?